lib/prefix.c

```diff
@@ -236,6 +236,8 @@ afi2str(afi_t afi)
 	return "IPv6";
     case AFI_ETHER:
 	return "ethernet";
+    case AFI_L2VPN:
```

Comment:
I think AFI_ETHER is something to be cleaned ( removed).
the 2 defines to keep:
AFI_L2VPN ( as per IANA)
AF_ETHERNET